sql/moz-fx-data-marketing-prod/acoustic/raw_recipient_v1/query.sql

```diff
@@ -13,6 +13,6 @@ SELECT
   LOWER(COALESCE(NULLIF(suppression_reason, ''), "unknown")) AS suppression_reason,
   submission_date,
 FROM
-  `moz-fx-data-marketing-prod.acoustic.raw_recipient_export_raw_v1`
+  `moz-fx-data-marketing-prod.acoustic.raw_recipient_raw_v1`
```

Comment:
I don't have context to understand the naming convention here. Why does `raw` appear twice in this name? Can you provide that context in descriptions in the relevant `metadata.yaml` files?

Reply:
I'll add this as a separate change next week when I'm back.

Reply:
Also, just to give some context here, raw_recipient is name of the actual type of object for which I pull data from Acoustic `raw_v1` is to represent this is raw data (raw layer) as coming from Acoustic.